lib/pltr/v2/helpers/hierarchy.js

```diff
@@ -61,8 +62,14 @@ export const newHierarchyLevel = (allHierarchyLevels) => {
     ...hierarchyLevel,
     name: nextLevelName(allHierarchyLevels.length),
     level: 0,
-    textColor: nextColor(allHierarchyLevels.length),
-    borderColor: nextColor(allHierarchyLevels.length),
     borderStyle: nextBorderStyle(allHierarchyLevels.length),
+    dark: {
```

Comment:
There's a slight problem here: this will cause files generated by mobile to break when opened in the desktop app.
We'll need to support both schemas for a while.
Here's a possible solution:
 - keep these keys here, and
 - copy the last set, setting, into these fields as well.